lib/redux/reducers/devices.js

```diff
@@ -25,6 +25,21 @@ const devices = {
     source: {type: 'device', driverId: 'Medtronic'},
     enabled: {mac: true, win: true}
   },
+  medtronic600: {
+    instructions: 'Connect your Bayer Contour Next Link 2.4 to your computer',
```

Comment:
`Contour Next Link 2.4` is the model: https://www.contournext.com/our-products/contour-next-products/contour-next-link-24-meter/